doc/releases/changelog-dev.md

```diff
@@ -420,59 +525,101 @@
   [2303, 23, 7674]
   ```
 
-* The number of `shots` can now be specified directly in QNodes as a standard keyword argument.
-  [(#8073)](https://github.com/PennyLaneAI/pennylane/pull/8073)
+* The :func:`~.noise.fold_global` transform is refactored to collect operators into a list directly rather than relying on queuing.
+  [(#8296)](https://github.com/PennyLaneAI/pennylane/pull/8296)
 
-  ```python
-  @qml.qnode(qml.device("default.qubit"), shots=1000)
-  def circuit():
-      qml.H(0)
-      return qml.expval(qml.Z(0))
-  ```
+<h4>Clifford+T decomposition</h4>
```

Comment:
I like the Clifford+T decomposition section above the Transforms section. Clifford+T is short and sweet and it's even mentioned in the Transforms section.